INSTALL.adoc

```diff
@@ -2,22 +2,33 @@
 
 == Prerequisites
 
-* The GNU C Compiler (gcc) is recommended, as the bytecode interpreter takes
+* A C Compiler is required.
+  The GNU C Compiler (`gcc`) is recommended as the bytecode interpreter takes
   advantage of GCC-specific features to enhance performance. gcc is the standard
-  compiler under Linux, OS X, and many other systems.
+  compiler under Linux, and many other systems.
+  However `clang` used in Mac OS, BSDs and others, also works fine.
+
+* GNU `make`, as well as GNU `awk` and GNU `sed` are required.
+  Alternatively, Dune can also be used but the support is experimental.
+
+* GNU `diff` is necessary to run the test suite.
+
+* The GNU BFD library is necessary for the `ocamlobjinfo` tool to be able
+  to process `.cmxs` files. (optional)
 
 * If you do not have write access to `/tmp`, you should set the environment
   variable `TMPDIR` to the name of some other temporary directory.
 
+== Prerequisites (special cases)
+
 * Under HP/UX, the GNU C Compiler (gcc), the GNU Assembler (gas), and GNU Make
   are all *required*.  The vendor-provided compiler, assembler and make tools
   have major problems.
 
-* Under Cygwin, the `gcc-core` and `make` packages are required.  `flexdll` is
-  necessary for shared library support.  `libX11-devel` is necessary for graph
-  library support and `libintl-devel` is necessary for the `ocamlobjinfo` tool
-  to be able to process `.cmxs` files.  `diffutils` is necessary to run the test
-  suite.
+* Under Cygwin, `flexdll` is necessary for shared library support.
+
+* Under Cygwin, `libintl-devel` is necessary for the `ocamlobjinfo` tool to be
+  able to process `.cmxs` files. (optional)
```

Comment:
This isn't correct, although in checking I believe I've found a bug in the handling of libbfd on Cygwin anyway. For now, this should be the `binutils` package, not `libintl-devel` (NB definitely `binutils` - Cygwin doesn't have `binutils-devel`)

Reply:
Actually, scratch that - having just double-checked it (there's no bug):
```suggestion
* Under Cygwin, `libintl-devel` and `zlib-devel` are necessary for the `ocamlobjinfo`
  tool to be able to process `.cmxs` files. (optional)
```
(given that `objinfo_helper.exe` is a 46M executable on my Cygwin installation, @nojb's patch to remove the dependency can't come quickly enough!)